libraries/standard/mqtt/integration-test/mqtt_system_test.c

```diff
@@ -1148,6 +1148,9 @@ void test_MQTT_Restore_Session_Complete_Incoming_Publish( void )
  */
 void test_MQTT_Resend_Unacked_Publish_QoS1( void )
 {
+    /* Start a persistent session with the broker. */
+    startPersistentSession();
```

Comment:
I think you can also replace the below calls to `OpenSSL_Connect()` and `establishMqttSession()` with `resumePersistentSession()` to assert if a session was actually resumed